content/en/docs/pluggable-components/overview.md

```diff
@@ -4,7 +4,7 @@ keywords: "kubernetes, docker, kubesphere, jenkins, istio, prometheus"
 description: "KubeSphere Release Notes For 3.0.0"
```

Comment:
The description should also be updated.

Reply:
This file is actually not finished yet as we do not have the resource consumption data. But sure, it won't hurt If I change the description now.